xml/http_servers.xml

```diff
@@ -1766,9 +1766,10 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
-   <fingerprint pattern="^PHP/(\S+)$">
-      <example>PHP/4.4.2-1build1</example>
-      <example>PHP/5.1.2</example>
+   <fingerprint pattern="^PHP/(\S+)(?:\s.+)?$">
+      <example service.component.version="4.4.2-1build1">PHP/4.4.2-1build1</example>
+      <example service.component.version="5.1.2">PHP/5.1.2</example>
+      <example service.component.version="5.3.14">PHP/5.3.14 ZendServer/5.0</example>
```

Comment:
Have an example of asserting a component of a component? Or is there a more sane-sounding way to do that?

The best thing I can think of otherwise is having 2 fingerprint matchers, but that requires the consumer to run the header string through the matchers twice...